tox.ini

```diff
@@ -9,30 +9,23 @@ envlist = py27,lint
 [base]
 deps =
     ipaddress
-    salt
+    salt<2017.0.0
 
 [testenv:py27]
-basepython = python2.7
```

Comment:
Why are you removing explicit py versions from here and from lint?

Reply:
Same as above.  It's not deletion as much as restoring to a known working state.